src/widgets/time-series/content-view.js

```diff
@@ -23,12 +25,24 @@ module.exports = cdb.core.View.extend({
     this.clearSubViews();
     this.$el.empty();
 
+    var sourceId = this._dataviewModel.get('source').id;
```

Comment:
Just a small comment, since this variables are only used in one place we could use them directly instead of declaring variables.